File: Final Project/Backend/Foddie_App/Restuarant/src/main/java/com/niit/backend/Restuarant/Domain/Restaurant.java
package com.niit.backend.Restuarant.Domain;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
@Document
public class Restaurant {
    @Id
    private int restId;
    private String email;
    private String restName;
    private String city;
    private byte[] url;
    private List<Menu>menuList;
}

File: Final Project/Frontend/src/app/Model-Classes/Restaurant.ts
import { FileHandle } from "./File-handle.model";
import { Menu } from "./Menu";

export class Restaurant{
    restId:number;
    restName:string;
    email:string;
    city:string;
    url:string;
    menuList:Menu[];
    selected:Boolean;
    constructor(){
        this.restId=0;
        this.restName="";
        this.email="";
        this.city="";
        this.url="";
        this.menuList=[];
        this.selected=false;
    }
}